specification/netapp/resource-manager/Microsoft.NetApp/stable/2019-10-01/netapp.json

```diff
@@ -0,0 +1,2710 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/volume"
+            }
+          },
+          "202": {
+            "description": "Accepted -- Create, update or delete request accepted; operation will complete asynchronously"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed."
+          }
+        },
+        "x-ms-examples": {
+          "Volumes_Update": {
+            "$ref": "examples/Volumes_Update.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Volumes"
+        ],
+        "operationId": "Volumes_Delete",
+        "summary": "Delete a volume",
+        "description": "Delete the specified volume",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroup"
+          },
+          {
+            "$ref": "#/parameters/AccountName"
+          },
+          {
+            "$ref": "#/parameters/PoolName"
+          },
+          {
+            "$ref": "#/parameters/VolumeName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted -- Create, update or delete request accepted; operation will complete asynchronously"
+          },
+          "204": {
+            "description": "NoContent -- Resource does not exist."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed."
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Volumes_Delete": {
+            "$ref": "examples/Volumes_Delete.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.NetApp/netAppAccounts/{accountName}/capacityPools/{poolName}/volumes/{volumeName}/breakReplication": {
+      "post": {
+        "tags": [
+          "Volumes Replication"
+        ],
+        "summary": "Break volume replication",
+        "description": "Break the replication connection on the destination volume",
+        "operationId": "Volumes_BreakReplication",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroup"
+          },
+          {
+            "$ref": "#/parameters/AccountName"
+          },
+          {
+            "$ref": "#/parameters/PoolName"
+          },
+          {
+            "$ref": "#/parameters/VolumeName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
```

Comment:
This API returns an async response code but is not marked as a long running operation via [x-ms-long-running-operation](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-long-running-operation)

Reply:
Marking this as a long running operation causes it to fail autorest validation. The reason for this is it wants also a terminal status code 200, 201 or 204. However the RP does not implement these response codes. If this means that we must introduce this into the RP then we will do so for the next API release. But meantime it looks like the swagger cannot be changed.